charts/gateway/templates/_helpers.tpl

```diff
@@ -63,3 +63,12 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create annotations for service
+*/}}
+{{- define "trino-gateway.serviceAnnotations" -}}
```

Comment:
What's the purpose of this template? We should expect correct values to be provided in the `valeus.yaml` file.

Reply:
Have removed the template.